miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebviewSpec.kt

```diff
@@ -162,17 +162,6 @@ class MiniAppWebviewSpec : BaseWebViewSpec() {
         miniAppWebView.settings.userAgentString shouldNotEndWith TEST_HA_NAME
     }
 
-    @Test
-    fun `when destroyView called then the MiniAppWebView should be disposed`() {
-        val displayer = Mockito.spy(miniAppWebView)
-        displayer.destroyView()
-
-        verify(displayer).stopLoading()
-        displayer.webViewClient shouldBe null
```

Comment:
Perhaps we just only need to remove this line?

Reply:
Oh, yeah you're right. I'll bring back this test.